### ISSUE TITLE
Training Curve for LibriSpeech speaker classification

### ISSUE THREAD

**Issue by toco2270853** [s3prl/s3prl#186]
Dear developer, I'm currently running tera 200k(with all transformation) checkpoint, and I found that using default config in the downstream task directory, the training accuracy seems not increase as fast as I think. t reaches about 82% accuracy for 10k step training(based on the 99.9% accuracy in paper, and the paper run for 24 epoch, I think it increase too slow).Can I get the training curve of yours for debugging?

**Comment by andi611** [comment]
Hi,
I'll try to find my training log, I'll update it here if I find it.
For the time being, 10K steps for 82% seems reasonable, the training set is large (approximately 90hrs of data), it will take some time. Give it some more updates (200k+ steps) and it should reach 99%.

**Comment by andi611** [comment]
This is the training curve of TERA "time + freq + mag", for your reference.

![image](https://user-images.githubusercontent.com/25531495/132984697-a6781079-829b-4b04-aa54-5330f16326b0.png)

Although I trained for 500k steps, apparently 200k is enough.
Here is the link to my training logs: [google drive](https://drive.google.com/drive/folders/1AubxrqHHZp9nG8KvVlLagXqO9XL_vGRx?usp=sharing)

**Comment by toco2270853** [comment]
Thank you so much!
